qiskit_ibm_runtime/batch.py

```diff
@@ -23,46 +23,54 @@
 class Batch(Session):
     """Class for running jobs in batch execution mode.
 
-    Similar to a ``session``, a Qiskit Runtime ``batch`` groups a collection of
-    iterative calls to the quantum computer. Batch mode can shorten processing time if all jobs
-    can be provided at the outset. To submit iterative jobs, use sessions instead.
+    The ``batch`` mode is designed to efficiently perform experiments that comprise multiple
+    independent jobs.
 
-    Using batch mode has these benefits:
+    Using the ``batch`` mode provides the following benefits:
         - The jobs' classical computation, such as compilation, is run in parallel.
           Thus, running multiple jobs in a batch is significantly faster than running them serially.
 
         - There is minimal delay between job, which can help avoid drift.
```

Comment:
We updated the wording in docs to say "There is usually minimal delay between jobs, which can help avoid drift", since other jobs, including system calibration, can run in between batch jobs.